### ISSUE TITLE
API Domain Attribute Download doesn't work anymore

### ISSUE THREAD

**Issue by chrissommer** [MISP/MISP#3453]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 16.04
| PHP version               | PHP 7.0.30-0ubuntu0.16.04.1
| MISP version / git hash   | 2.4.93

### Expected behavior
curl -X GET -k -H "Authorization: APIKEY" https://MISP-IP/attributes/text/download/domain/ - returns list of Domains

### Actual behavior
curl -X GET -k -H "Authorization: APIKEY" https://MISP-IP/attributes/text/download/domain/ - returns Internal Server Error

### Steps to reproduce the behavior
Go to the CLI of your MISP and enter:
curl -X GET -k -H "Authorization: APIKEY" https://MISP-IP/attributes/text/download/domain/

### Logs, screenshots, configuration dump, ...
All Other API Exports are working fine (IPs, Hashes, URLs, Hostnames, ...)
Domain Export is delivering an internal Server Error since a few days (maybe since Update to 2.4.93)

**Comment by SteveClement** [comment]
I cannot reproduce this: 

```
$ curl -X GET -k -H "Authorization: AUTHKEY" https://misp.local/attributes/text/download/domain/
attacker.com
foo.be
localhost.lu
```

@csommerkapsch can you check: /var/www/MISP/app/tmp/logs
and other logs to see what the internal server error is so we can investigate.

How many results are you expecting? Hundreds? Thousands? Millions?
Are there any special domains, like emoji domains or other internationalized domain names.?

**Comment by chrissommer** [comment]
Hi Steve,

I am expecting 20k domains.

I saw in the /var/www/MISP/app/tmp/logs/error.log, that MISP thinks there is not enogh Disk Space, although there were 1,5 GB left.
Error: [PDOException] SQLSTATE[HY000]: General error: 3 Error writing file '/tmp/MYYxdSr7' (Errcode: 28 "No space left on device") 

I increased the Diskspace and now everthing is working fine.

Thank you for your Support

BR
Chris
